opensrp-core/src/main/java/org/smartregister/repository/AllSharedPreferences.java

```diff
@@ -398,5 +398,13 @@ public void updateLastAuthenticationHttpStatus(int httpStatus) {
     public int getLastAuthenticationHttpStatus() {
         return preferences.getInt(LAST_AUTHENTICATION_HTTP_STATUS, 0);
     }
+
+    public void saveUserId(String username, String userId) {
+        preferences.edit().putString(AllConstants.USER_ID_PREFIX + username, userId).apply();
+    }
+
+    public String getUserId(String userName) {
+        return StringUtils.isNotBlank(userName) ? preferences.getString(AllConstants.USER_ID_PREFIX + userName, null) : "";
+    }
```

Comment:
@junaidwarsivd What does the `user id` do? Why are we concatenating it with the `username`?

Reply:
@dubdabasoduba  these changes are from the https://github.com/opensrp/opensrp-client-core/pull/887/files merge  as it has resolved codacy issues